docs/antrea-ipam.md

```diff
@@ -31,7 +31,8 @@ NodeIPAM dictionary contains the following items:
 controller. Default is false.
 
 - `clusterCIDRs`: CIDR Ranges for Pods in cluster. String array containing single
-CIDR range, or multiple ranges. The CIDRs could be either IPv4 or IPv6.
+CIDR range, or multiple ranges. The CIDRs could be either IPv4 or IPv6. Example
+values are like: `[172.100.0.0/16]`, `[172.100.0.0/20 172.100.32.0/20 fd00:172:100::/64]`.
```

Comment:
shouldn't values in a YAML array be comma-separated?

Reply:
s/Example values are like:/ Example values:

Reply:
Fixed them.

Reply:
I think that the IPs should be quoted, to be valid for yaml. But I'm not sure about. Otherwise LGTM.

Reply:
I tested it. It does work without "".